File: CapaEntidad/src/Entidades/Historia.java
package Entidades;

import java.sql.Date;

public class Historia {
	private int numhistoria;
	private String apepat;
	private String apemat;
	private String nombres;
	private Date fechanac;
	private String dni;
	private String direccion;
	public void setNumhistoria(int numhistoria) {
		this.numhistoria = numhistoria;
	}
	public void setApepat(String apepat) {
		this.apepat = apepat;
	}
	public void setApemat(String apemat) {
		this.apemat = apemat;
	}
	public void setNombres(String nombres) {
		this.nombres = nombres;
	}
	public void setFechanac(Date fechanac) {
		this.fechanac = fechanac;
	}
	public void setDni(String dni) {
		this.dni = dni;
	}
	public void setDireccion(String direccion) {
		this.direccion = direccion;
	}
	
	

}

File: escritorio/Escritorio/Escritorio/Historia.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Escritorio
{
    public class Historia
    {
        private int numhistoria;
        private String apepat;
        private String apemat;
        private String nombres;
        private String fechanac;
        private String dni;
        private String direccion;

        public int Numhistoria
        {
            get
            {
                return numhistoria;
            }

            set
            {
                numhistoria = value;
            }
        }

        public string Apepat
        {
            get
            {
                return apepat;
            }

            set
            {
                apepat = value;
            }
        }

        public string Apemat
        {
            get
            {
                return apemat;
            }

            set
            {
                apemat = value;
            }
        }

        public string Nombres
        {
            get
            {
                return nombres;
            }

            set
            {
                nombres = value;
            }
        }

        public string Fechanac
        {
            get
            {
                return fechanac;
            }

            set
            {
                fechanac = value;
            }
        }

        public string Dni
        {
            get
            {
                return dni;
            }

            set
            {
                dni = value;
            }
        }

        public string Direccion
        {
            get
            {
                return direccion;
            }

            set
            {
                direccion = value;
            }
        }
    }
}
